docs/README.md

```diff
@@ -109,6 +109,8 @@ kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/release-1
 ```
 
 #### Deploy driver
+Latest master branch only supports driver deployment on kubernetes version >= 1.17
```

Comment:
Can we make this more generic? I'm thinking something like "Please see the compatibility matrix above before you deploy the driver". So we don't have to keep updating this part as well.

Reply:
Updated